includes/hocr.inc

```diff
@@ -583,7 +583,7 @@ class HOCR {
    */
   public static function stripDoctype($hocr_string) {
     // Wipe out the HTML DOCTYPE definition.
-    $doctype_regex = '/<!DOCTYPE\s+html.*?>\s+/si';
+    $doctype_regex = '/<!DOCTYPE\s+html.*?>+/si';
```

Comment:
If you don't have the `\s` for spaces then the `+` transfers to the `>`. Is that by design?